ruby/BUILD.bazel

```diff
@@ -19,6 +21,46 @@ ruby_library(
     ],
 )
 
+java_export(
+    name = "protobuf_java_release",
+    maven_coordinates = "com.google.protobuf:protobuf-jruby:%s" % PROTOBUF_RUBY_VERSION,
+    srcs = glob(["src/main/java/com/google/protobuf/jruby/*"]),
+    runtime_deps = [
+        "@maven//:org_jruby_jruby_complete",
+        "@com_google_protobuf//java/util",
+    ],
+)
+
+filegroup(
+  name = "protobuf_java_jar",
+  srcs = [":protobuf_java_release-project"],
+)
+
+# Run with the flag: --define=ruby_platform=c
+sh_binary(
+    name = "jruby_release",
+    data = [
+      ":protobuf_java_jar",
+      "//:well_known_ruby_protos",
+    ],
+    srcs = [
+      "build_jruby_release.sh",
+    ],
+)
+
+# Run with the flag: --define=ruby_platform=java
+sh_binary(
+    name = "release",
```

Comment:
What's the point of this target?  It looks like it should be broken, as it's missing dependencies on a lot of files it uses.  Also, this looks like it's meant for jruby, where is the cruby version?

Reply:
The tags got switched, this is for cruby and I left it as the default because that's how the other targets are named.

When I run it it builds a gem that installs, why do you think it should be broken?

Reply:
I see what you're saying, I don't have a formal dependency on the source files since they are in this directory I just paste the things that need to be generated in the folder.

I'm looking at how to get the bazel data files and that's more inconvenient than just using the directory structure that already exists here.

Do you have any better ideas on how to do this?

Reply:
Are you using files outside of the sandbox then?  Ideally you would add them as formal dependencies in the BUILD.bazel file and then use the relative paths to get the sandboxed versions.  inline_sh_binary is a pretty useful tool for getting bazel variable expansion in sh files

Reply:
I fixed this to depend on all files we need and run in bazel-bin instead of the github repo itself.